Tests/SentryTests/SentryCrash/SentryThreadInspectorTests.swift

```diff
@@ -37,6 +40,41 @@ class SentryThreadInspectorTests: XCTestCase {
         XCTAssertTrue(30 < stacktrace?.frames.count ?? 0, "Not enough stacktrace frames.")
     }
     
+    @available(macOS 10.12, iOS 10.0, tvOS 10.0, *)
+    func testStacktraceHasFrames_forEveryThread() {
+        SentrySDK.start { $0.stitchAsyncCode = true }
```

Comment:
`m`: Can we create two tests one with `stitchAsyncCode = true` and one without, please?